<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QtArc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="address"/>
    </item>
    <item>
     <widget class="QListView" name="treeView">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="movement">
       <enum>QListView::Free</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_FileNew"/>
    <addaction name="action_FileOpen"/>
    <addaction name="action_FileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_FileProperties"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_EditCut"/>
    <addaction name="action_EditCopy"/>
    <addaction name="action_EditPaste"/>
    <addaction name="separator"/>
    <addaction name="action_EditSelectAll"/>
    <addaction name="action_EditDeselect"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_HelpAbout"/>
    <addaction name="action_HelpAboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="action_AddFile"/>
    <addaction name="action_AddDirectory"/>
    <addaction name="action_Extract"/>
    <addaction name="action_Delete"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_ViewTree"/>
    <addaction name="action_ViewTable"/>
    <addaction name="action_ViewIcons"/>
    <addaction name="action_ViewColumns"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Actions"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_FileNew"/>
   <addaction name="action_FileOpen"/>
   <addaction name="separator"/>
   <addaction name="action_AddFile"/>
   <addaction name="action_AddDirectory"/>
   <addaction name="action_Extract"/>
   <addaction name="action_Delete"/>
   <addaction name="separator"/>
   <addaction name="action_Up"/>
  </widget>
  <action name="action_FileNew">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-new.png</normaloff>:/icons/icons/24x24/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_FileOpen">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-open.png</normaloff>:/icons/icons/24x24/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_FileSaveAs">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-save-as.png</normaloff>:/icons/icons/24x24/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Extract">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/extract-archive.png</normaloff>:/icons/icons/24x24/extract-archive.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Extract</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_FileProperties">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-properties.png</normaloff>:/icons/icons/24x24/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Properties</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_AddFile">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/add-files-to-archive.png</normaloff>:/icons/icons/24x24/add-files-to-archive.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;File</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+F</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_AddDirectory">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/add-folder-to-archive.png</normaloff>:/icons/icons/24x24/add-folder-to-archive.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;Directory</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_EditCut">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-cut.png</normaloff>:/icons/icons/24x24/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_EditCopy">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-copy.png</normaloff>:/icons/icons/24x24/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_EditPaste">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-paste.png</normaloff>:/icons/icons/24x24/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Delete">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-delete.png</normaloff>:/icons/icons/24x24/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string notr="true">Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_HelpAbout">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/help-about.png</normaloff>:/icons/icons/24x24/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_HelpAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="action_EditSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_EditDeselect">
   <property name="text">
    <string>Deselect</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_ViewTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>T&amp;ree</string>
   </property>
  </action>
  <action name="action_ViewTable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Table</string>
   </property>
  </action>
  <action name="action_ViewIcons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Icons</string>
   </property>
  </action>
  <action name="action_ViewColumns">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Columns</string>
   </property>
  </action>
  <action name="action_Up">
   <property name="icon">
    <iconset resource="../pyqtarc.qrc">
     <normaloff>:/icons/icons/24x24/go-up.png</normaloff>:/icons/icons/24x24/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../pyqtarc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>Main</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
